allensdk/brain_observatory/ecephys/ecephys_session_api/ecephys_nwb_session_api.py

```diff
@@ -168,8 +168,8 @@ def _get_full_units_table(self) -> pd.DataFrame:
             ]
             units.drop(columns=["quality"], inplace=True)
 
-        units = units[units["amplitude_cutoff"] <= self.amplitude_cutoff_maximum]
```

Comment:
If we are not applying this filter anymore, we need to yank it globally -including from the project-level accessors

Reply:
Oh I see, you are using the session object for metrics calculations and want metrics for all units. In that case set these filter values during construction of the nwb api object

Reply:
Looks like an accidental checkin, I'll put in back.

Reply:
I don't think this is accidental. I think the goal was to make sure that when the stimulus analysis module is run in production it computes metrics for all units.

Reply:
I think there needs to be a flag in the EcephysSession object that allows the user to get the full units table; and the stimulus analysis module can use this to compute metrics for all units.

Reply:
@jsiegle There is. It is in the constructor of the nwb api (you can pass kwargs when you build the session from_lims). That is the correct way to get this behavior.

Reply:
e.g:
```
import numpy as np
from allensdk.brain_observatory.ecephys.ecephys_session import EcephysSession

path = "/data/dynamic-brain-workshop/visual_coding_neuropixels/2019/session_762602078/session_762602078.nwb"
ssn = EcephysSession.from_nwb_path(path, api_kwargs={
    "amplitude_cutoff_maximum": np.inf,
    "presence_ratio_minimum": -np.inf,
    "isi_violations_maximum": np.inf,
    "filter_by_validity": False # actually you probably still want this one
})
print(ssn.units.shape)

ssn2 = EcephysSession.from_nwb_path(path)
print(ssn2.units.shape)
```
I just ran this on aws and got 2159 vs 554, which seems large. Maybe the defaults should be looser (or maybe this session has a bunch of dubious units)